README.md

```diff
@@ -1,2 +1,77 @@
 # fetch
```

Comment:
I did a few minutes of research too and didn't find anything that was confusingly similar. I did find https://git-scm.com/docs/git-fetch, which seems like an appropriate play. I wondered if we should call it github-fetch, but then I thought someday it'd be easy enough to add support for bitbucket, gitlab, etc.